File: fix_encoding.py
# -*- coding: utf-8 -*-
import codecs

# Читаем файл с неправильной кодировкой и исправляем
with open('lib/servicesData.ts', 'r', encoding='utf-8') as f:
    content = f.read()

# Сохраняем с правильной кодировкой
with codecs.open('lib/servicesData.ts', 'w', encoding='utf-8-sig') as f:
    f.write(content)

print("Encoding fixed!")

File: scripts/fix-encoding.js
const fs = require('fs');
const path = require('path');

// Read the current file
const filePath = path.join(__dirname, '..', 'lib', 'servicesData.ts');
let content = fs.readFileSync(filePath, 'utf8');

// Fix the garbled Russian text by replacing with correct values
// The issue is that PowerShell saved the file with wrong encoding

// Let's just re-save it properly with UTF-8
fs.writeFileSync(filePath, content, { encoding: 'utf8' });

console.log('File re-saved with proper UTF-8 encoding');
